src/components/farm-info-card/FarmOverview.tsx

```diff
@@ -0,0 +1,195 @@
+    console.log("withdraw")
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    await masterchefContract.withdrawAll()
+  }
+
+  async function onClickDeposit() {
+    console.log("deposit")
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    await masterchefContract.depositAll()
+  }
+
+  async function onClickClaim() {
+    console.log("claim")
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    await masterchefContract.claim()
+  }
+  console.log({ tokens: formattedData.tokens })
+  return (
+    <div
+      className={classNames("poolOverview", {
+        outdated: isOutdated || shouldMigrate,
+      })}
+    >
+      <div className="left">
+        <div className="titleAndTag">
+          <h4 className="title">{formattedData.name}</h4>
+          {(shouldMigrate || isOutdated) && <Tag kind="warning">OUTDATED</Tag>}
+          {poolData.isPaused && <Tag kind="error">PAUSED</Tag>}
+        </div>
+        {hasShare && (
+          <div className="balance">
+            <span>{t("balance")}: </span>
+            <span>{`$${formattedData.userBalanceUSD}`}</span>
+          </div>
+        )}
+        <div className="tokens">
+          <span style={{ marginRight: "8px" }}>[</span>
+          {formattedData.tokens.map(({ symbol, icon }) => (
+            <div className="token" key={symbol}>
+              <img alt="icon" src={icon} />
+              <span>{symbol}</span>
+            </div>
+          ))}
+          <span style={{ marginLeft: "-8px" }}>]</span>
+        </div>
+      </div>
+
+      <div className="right">
+        <div className="poolInfo">
+          {formattedData.apy && (
+            <div className="margin">
+              <span className="label">{`${t("apy")}`}</span>
+              <span>{formattedData.apy}</span>
+            </div>
+          )}
+          <div className="margin">
+            <span className="label">TVL</span>
+            <span>{`$${formattedData.reserve}`}</span>
+          </div>
+          {/*formattedData.volume && (
+            <div>
+              <span className="label">{`${t("24HrVolume")}`}</span>
+              <span>{formattedData.volume}</span>
+            </div>
+          )*/}
+        </div>
+        <div className="buttons">
+          <Link to={`farm/withdraw`}>
+            <Button kind="secondary">{t("withdraw")}</Button>
+          </Link>
+          {shouldMigrate ? (
+            <Button
+              kind="temporary"
+              onClick={onClickMigrate}
+              disabled={!hasShare}
+            >
+              {t("migrate")}
+            </Button>
+          ) : (
+            <Link to={`farm/deposit`}>
+              <Button
+                kind="primary"
+                disabled={poolData?.isPaused || isOutdated}
+              >
+                {t("deposit")}
+              </Button>
+            </Link>
+          )}
+        </div>
+        {/* <span style={{ marginTop: "8px" }}></span> */}
```

Comment:
Avoid pushing commented out code, if you do because you have a reason, you need to add a description for why is commented out.